docs/data/debugger.md

```diff
@@ -3,20 +3,36 @@ title: Debugging Analytics
 description: Troubleshoot and debug any data errors that may occur in the instrumentation process.
 ---
 
-Data validation is a critical step in the instrumentation process. Amplitude lets validate your event data via Amplitude's [User Lookup](#user-lookup) or using the [Instrumentation Explorer](#instrumentation-explorer) Chrome extension.
+Data validation is a critical step in the instrumentation process. Amplitude lets validate your event data by:
```

Comment:
Amplitude lets whom?

Reply:
Added "you" and changed it to "Amplitude lets you validate your event data by:"